src/redux/Categories/Categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'checkStatus';
```

Comment:
- [ ] It 's a good thing you implemented the changes requested by the previous code reviewer, but as a requirement for this project and best practice, please we are supposed to use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) to define our action types like so(just an example); `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.

Reply:
Thank you for your review. I have solved this issue.